lib/routes/xmu/kydt.ts

```diff
@@ -0,0 +1,68 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/kydt',
+    categories: ['university'],
+    example: '/xmu/kydt',
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['soe.xmu.edu.cn/kxyj/kydt.htm'],
+        },
+    ],
+    name: '科研动态',
+    maintainers: ['linsenwang'],
+    handler,
+};
+
+async function handler() {
+    const host = `https://soe.xmu.edu.cn/kxyj/kydt.htm`;
```

Comment:
```suggestion
    const host = 'https://soe.xmu.edu.cn/kxyj/kydt.htm';
```
Only use template literal when necessary.